core/config/project_settings.cpp

```diff
@@ -291,31 +291,26 @@ bool ProjectSettings::_set(const StringName &p_name, const Variant &p_value) {
 			return true;
 		}
 
-		if (!disable_feature_overrides) {
+		{ // Feature overrides.
 			int dot = p_name.operator String().find(".");
 			if (dot != -1) {
 				Vector<String> s = p_name.operator String().split(".");
 
-				bool override_valid = false;
 				for (int i = 1; i < s.size(); i++) {
 					String feature = s[i].strip_edges();
-					if (OS::get_singleton()->has_feature(feature) || custom_features.has(feature)) {
-						override_valid = true;
-						break;
+					Pair<StringName, StringName> fo(feature, p_name);
+
+					if (!feature_overrides.has(p_name)) {
```

Comment:
Shouldn't this check for `s[0]` instead of `p_name`?

Which is btw used often enough in this for loop that I think it could be factored out in a local variable with a clearer name.

Reply:
Right, @bruvzg made the same comment above, thanks for pointing out!